docs/api/puppeteer.locator.wait.md

```diff
@@ -0,0 +1,27 @@
+---
+sidebar_label: Locator.wait
+---
+
+# Locator.wait() method
+
+Waits for the locator to get the serialized value from the page.
```

Comment:
Please update https://pptr.dev/guides/locators with an example on how to use this

Reply:
I will do this in a separate PR because as of now, `wait` doesn't do much without `map`.